neurovault/apps/statmaps/static/scripts/metadata-edit.js

```diff
@@ -0,0 +1,382 @@
+/*global window, document, jQuery, Handsontable, NVMetadata */
+(function ($) {
+  'use strict';
+
+  function isString(val) {
+    return (typeof val === 'string' || val instanceof String);
+  }
+
+  function cache() {
+    var storage = {};
+    return {
+      get: function (key) {
+        return storage[key];
+      },
+      set: function (key, value) {
+        storage[key] = value;
+      }
+    }
+  }
+
+  function debounce(func, wait, immediate) {
+    var timeout;
+    return function () {
+      var context = this,
+        args = arguments;
+      var later = function () {
+        timeout = null;
+        if (!immediate) func.apply(context, args);
+      };
+      var callNow = immediate && !timeout;
+      clearTimeout(timeout);
+      timeout = setTimeout(later, wait);
+      if (callNow) func.apply(context, args);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'func'.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/825092758/issues/source?bid=2767031&fileBranchId=2767194#l33)